install/image-references

```diff
@@ -58,3 +58,7 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-kube-rbac-proxy:4.2.0
+  - name: ovirt-machine-controllers
+    from:
+      kind: DockerImage
+      name: quay.io/rgolangh/origin-ovirt-machine-controllers
```

Comment:
we need to update this to point to the openshift canonical repo?

Is there an ART image to match this? otherwise this will break CI

Reply:
I sent this https://github.com/openshift/release/pull/6151 for it.